params/protocol_params.go

```diff
@@ -148,38 +149,41 @@ const (
-	LocalDurationLimit                = big.NewInt(1) // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
-	TimeToStartTx              uint64 = 5 * BlocksPerDay
-	BlocksPerDay               uint64 = new(big.Int).Div(big.NewInt(86400), DurationLimit).Uint64() // BlocksPerDay is the number of blocks per day assuming 12 second block time
-	DifficultyAdjustmentPeriod        = big.NewInt(720)                                             // This is the number of blocks over which the average has to be taken
-	DifficultyAdjustmentFactor int64  = 40                                                          // This is the factor that divides the log of the change in the difficulty
-	MinQuaiConversionAmount           = new(big.Int).Mul(big.NewInt(10000000000), big.NewInt(GWei)) // 0.000000001 Quai
-	MaxWorkShareCount                 = 16
-	WorkSharesThresholdDiff           = 3 // Number of bits lower than the target that the default consensus engine uses
-	WorkShareP2PThresholdDiff         = 7 // Default value in bits lower than the target
-	WorkSharesInclusionDepth          = 3 // Number of blocks upto which the work shares can be referenced and this is protocol enforced
-	LockupByteToBlockDepth            = make(map[uint8]uint64)
-	LockupByteToRewardsRatio          = make(map[uint8]*big.Int)
-	ExchangeRate                      = big.NewInt(86196385918997143) // This is the initial exchange rate in Qi per Quai in Its/Qit // Garden = big.NewInt(166666666666666667)
+	MaxGossipsubPacketSize             = 3 << 20
+	GasCeil                     uint64 = 30000000
+	ColosseumGasCeil            uint64 = 30000000
+	GardenGasCeil               uint64 = 30000000
+	OrchardGasCeil              uint64 = 30000000
+	LighthouseGasCeil           uint64 = 30000000
+	LocalGasCeil                uint64 = 30000000
+	DurationLimit                      = big.NewInt(5) // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
+	GardenDurationLimit                = big.NewInt(5) // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
+	OrchardDurationLimit               = big.NewInt(5) // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
+	LighthouseDurationLimit            = big.NewInt(5) // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
+	LocalDurationLimit                 = big.NewInt(1) // The decision boundary on the blocktime duration used to determine whether difficulty should go up or not.
+	TimeToStartTx               uint64 = 5 * BlocksPerDay
+	BlocksPerDay                uint64 = new(big.Int).Div(big.NewInt(86400), DurationLimit).Uint64() // BlocksPerDay is the number of blocks per day assuming 5 second block time
```

Comment:
BlocksPerDay should probably be dynamic on the target time. Right now its hardcoded to assume 5s target

Reply:
for that we need to be using the config to know which env we are running and make a function, i can do it, since we made all the duration limits 5, is it fine to keep this for now and i can create an issue to do that in the future